components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/endpoint/registry/constants/EndpointRegistryConstants.java

```diff
@@ -69,5 +69,14 @@
     public static final String DEFAULT_REGISTRY_NAME = "Default Registry";
 
     public static final String SYSTEM_USER_NAME = "SYSTEM_USER";
+
+    public static final String DEFINITION_TYPE_OAS = "OAS";
+
+    public static final String DEFINITION_TYPE_WSDL1 = "WSDL1";
+
+    public static final String DEFINITION_TYPE_WSDL2 = "WSDL2";
+
+    public static final String DEFINITION_TYPE_GQL_SDL = "GQL_SDL";
+
```

Comment:
Do we need these spaces between every line?

Reply:
Fixed in https://github.com/wso2-enterprise/service-catalog/pull/73